srcpkgs/gzdoom/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'gzdoom'
 pkgname=gzdoom
 version=4.3.3
-revision=1
+revision=2
 _tagdate=2019-07-09
 wrksrc="${pkgname}-g${version}"
 build_style=cmake
 configure_args="-DINSTALL_PK3_PATH=share/gzdoom"
 hostmakedepends="pkg-config tar xz"
-makedepends="SDL2-devel gtk+3-devel fluidsynth-devel libgme-devel libgomp-devel ppl-devel"
-depends="libfluidsynth gtk+3"
+makedepends="SDL2-devel gtk+3-devel fluidsynth-devel libgme-devel libgomp-devel
+ ppl-devel libopenal-devel libsndfile-devel"
+depends="libfluidsynth gtk+3 libopenal libsndfile"
```

Comment:
Huh, are all of these not auto-detected?

Reply:
No, but I'm also not sure this PR should be open. I get sound with the old version as well.

Reply:
So, removing `libopenal` from the system makes sound stop working, but it doesn't seem to matter if it's included in the build. I hate dlopen'd stuff.

Reply:
Yeah, it's some stupid-ass dlopen thing. Since this is an [upstream recommendation](https://zdoom.org/wiki/Compile_GZDoom_on_Linux#Install_dependencies), I will leave them there.